authlib/jose/rfc7518/jwe_algs.py

```diff
@@ -187,6 +213,15 @@ def prepare_key(self, raw_data):
             return raw_data
         return ECKey.import_key(raw_data)
 
+    def generate_preset(self, enc_alg, key):
+        epk = key.generate_key(key['crv'], is_private=True)
```

Comment:
Should we call self._generate_ephemeral_key(key) instead?

Reply:
Fixed.